src/components/Panel/Panel.tsx

```diff
@@ -4,13 +4,18 @@ import styled from "styled-components";
 export type PanelPadding = "none" | "xs" | "sm" | "md" | "lg" | "xl";
 export type PanelColor = "default" | "muted" | "transparent";
 
+type Orientation = "horizontal" | "vertical";
 export interface PanelProps extends HTMLAttributes<HTMLDivElement> {
   hasBorder?: boolean;
   hasShadow?: boolean;
   color?: PanelColor;
   padding?: PanelPadding;
   children?: React.ReactNode;
-  orientation?: "horizontal" | "vertical";
+  orientation?: Orientation;
+  width?: string;
+  fillWidth?: boolean;
+  height?: string;
+  fillHeight?: boolean;
```

Comment:
I think we can directly add
`fillHeight?: true`

Reply:
no i think we want to keep the default behavior of a standard div since this panel will be used in lots of cases

Reply:
@serdec What I meant was about the typescript
In this case
<Panel fillHeight /> or <Panel /> is allowed but <Panel fillHeight={false}/> is not allowed 
Adding =false is visually not appealing and since the default is already undefined it won't match as true